DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -108,7 +108,8 @@ public static void readParquetFile(String filePath) throws IOException {
 
     @FunctionalInterface
     public interface ColumnDefinitionConsumer {
-        void accept(String name, Class<?> dataType, Class<?> componentType, boolean isGroupingColumn, String codecName, String codecArgs);
+        // objectWidth == -1 means not present.
```

Comment:
This should be formatted as a JavaDoc. It's 10x more useful than as a single-line comment like this.